communication/socket/tcp/create-tcp-socket.yml

```diff
@@ -11,11 +11,13 @@ rule:
     examples:
       - Practical Malware Analysis Lab 01-01.dll_:0x10001010
   features:
-    - and:
-      - number: 6 = IPPROTO_TCP
-      - number: 1 = SOCK_STREAM
-      - number: 2 = AF_INET
-      - or:
-        - api: ws2_32.socket
-        - api: ws2_32.WSASocket
-        - api: socket
+    - or:
```

Comment:
Don't forget to update the `authors` field

Reply:
Noted. https://github.com/mandiant/capa-rules/pull/601/commits/b3cd0abc6de717aaedda04b51804f1c562c8b472 updates the field.